.circleci/config.yml

```
@@ -193,6 +193,7 @@ workflows:
           requires:
             - build
       - ship/node-publish:
+          publish-command: npm publish --tag first-availability
```

Curious if we should publish using the `fa` tag instead of `first-availability`, with flutter we also used fa, probably easier for the user.

```suggestion
          publish-command: npm publish --tag fa
```